### ISSUE TITLE
9.1 I'm not sure this belongs here

### ISSUE THREAD

**Issue by payne-chris-r** [sarahholden/fundamentals#225]
I'm not sure any of this belongs in fundamentals, period. The difference between function declaration and expression are very subtle, and actually go away once we start using ECMA6 syntax. I understand why it's here, I just really think it's a waste of time for beginners that are a) trying to understand javascript, and b) developers who will shortly not have to worry about it (and in fact have it work the opposite way). 

I don't know. I just think it's unimportant to learning HOW and WHY to create/invoke functions, and may be better ignored and addressed at a later date (if at all). 

This is the section I'm talking about, I think it's ok to remove all of it.
```
Other Ways to Define Your Functions

Although we didn't mention it above, when we define a function using the following syntax, this is called a function expression:

var errorAlert = function () {
    alert("Please be sure to fill out all required fields.");
};
Function expressions are only one way to define a function in JavaScript. Another common method is a function declaration, which looks like this:

function errorAlert () {
    alert("Please be sure to fill out all required fields.");
};
You'll notice that there are only a couple minor differences in the way that function expressions and function declarations are written out.

Hoisting

Let's dive a little deeper into one of the primary differences between function declarations and function expressions.

We'll be using function expressions for the remainder of Fundamentals, but it's important to have a high-level understanding of how the two differ and how that may impact your code.

While both methods are similar, one of their main differences is the concept of hoisting.

In JS, function declarations are always moved, or "hoisted," to the top of their scope by the interpreter. (Remember, the interpreter is the console in JS — the software that runs the code).

In other words, you can call a function declaration before defining it.

Let's look at an example of hoisting:



Since we'll be using function expressions from here on out, it's important to remember that you must define a function before you can call it.
```
